### PR TITLE
test_pvc_multi_clone_performance.py

### PR DESCRIPTION
**Pull request by ypersky1980**
test_pvc_multi_clone_performance.py is a new implementation for multi clones performance test, which consumes low memory resources.

### CODE REVIEW

File: tests/e2e/performance/test_pvc_multi_clone_performance.py [hunk 1]
```diff
@@ -0,0 +1,388 @@
...
+        for sublog in logs:
+            for line in sublog:
+                if st is None and "provision" in line and clone_name in line and "started" in line:
+                    st = line.split(" ")[1]
+                    st = datetime.datetime.strptime(st, format)
+                elif et is None and "provision" in line and clone_name in line and "succeeded" in line:
+                    et = line.split(" ")[1]
+                    et = datetime.datetime.strptime(et, format)
+                if (st is not None and et is not None):
+                    break;
```
Comment (jilju) [inline tests/e2e/performance/test_pvc_multi_clone_performance.py:376]: Remove semicolon

File: tests/e2e/performance/test_pvc_multi_clone_performance.py [hunk 2]
```diff
@@ -0,0 +1,388 @@
...
+            if (st is not None and et is not None):
+                break;
```
Comment (jilju) [inline tests/e2e/performance/test_pvc_multi_clone_performance.py:378]: Remove semicolon

File: tests/e2e/performance/test_pvc_multi_clone_performance.py [hunk 3]
```diff
@@ -0,0 +1,388 @@
...
+            kubeconfig = cluster_dir_kubeconfig
+        else:
+            kubeconfig = None
+
+        command = (f'oc --kubeconfig {kubeconfig} -n {namespace} {cmd}')
+        return self.run_command(command)
+
+    def measure_clone_creation_time(self, clone_name, start_time):
+        """
+
```
Comment (jilju) [inline tests/e2e/performance/test_pvc_multi_clone_performance.py:346]: Remove extra blank line

Comment (ypersky1980) [reply tests/e2e/performance/test_pvc_multi_clone_performance.py]: Fixed and added docstring.

File: tests/e2e/performance/test_pvc_multi_clone_performance.py (hunk 2 above)
Comment (ypersky1980) [reply tests/e2e/performance/test_pvc_multi_clone_performance.py:378]: Done, will be pushede in the next commit. Thank you!

**Comment by vasukulkarni** [comment]
@ypersky1980  there are quite a bit of merge commits in that PR, I will suggest that you take a new master, cut a branch, copy your code changes,  add new commit and push that change

File: tests/e2e/performance/test_pvc_multi_clone_performance.py (hunk 1 above)
Comment (ypersky1980) [reply tests/e2e/performance/test_pvc_multi_clone_performance.py:376]: @jilju Done.

File: tests/e2e/performance/test_pvc_multi_clone_performance.py [hunk 4]
```diff
@@ -0,0 +1,314 @@
+import os
+import logging
+from ocs_ci.ocs.resources import pod
+from ocs_ci.ocs import constants, exceptions
+from ocs_ci.ocs.cluster import CephCluster
+from ocs_ci.utility.utils import ocsci_log_path
+from ocs_ci.framework.testlib import (
+    skipif_ocs_version, skipif_ocp_version, E2ETest, performance
+)
+import yaml
```
[the remark this answers is not in the document]
Comment (ypersky1980) [reply tests/e2e/performance/test_pvc_multi_clone_performance.py:10]: @jilju  Thank you, done!

**Comment by ypersky1980** [comment]
Closing this PR following Vasu's  request to open a new one with less merges.

**Pull request closed without merging**